doc/man1/flux-cron.rst

```diff
@@ -6,182 +6,262 @@ flux-cron(1)
-   sync-event, the *stats.deferred* statistic is incremented.
-
-**interval** [OPTIONS] *interval* *command*
-   Create a cron entry to execute *command* every *interval*, where *interval*
-   is an arbitrary floating point duration with optional suffix *s* for
-   seconds, *m* for minutes, *h* for hours and *d* for days.
-   Options:
+The following sub-commands are available:
 
-.. program:: flux cron interval
+tab
+---
 
-.. option:: -n, --name=STRING
+.. program:: flux cron tab
 
-      Set a name for this cron entry to *STRING*.
+Process one or more lines containing crontab expressions from *file*
+(stdin by default).  Each valid crontab line will result in a new cron
+entry registered with the Flux cron service.
 
-.. option:: -a, --after=TIME
+Crontab lines have five standard fields, similar to :linux:man5:`crontab`:
 
-      The first task will run after a delay of *TIME* instead of *interval*.
-      After the first task the entry will continue to execute every *interval*.
+.. list-table::
 
-.. option:: -c, --count=N
+   * - minutes
+     - 0-59
 
-      The entry will be run a total of *N* times, then stopped.
+   * - hours
+     - 0-23
 
-.. option:: -o, --options=LIST
+   * - day of month
+     - 0-23
+
+   * - month
+     - 0-11
+
+   * - day of week
+     - 0-6
+
+Everything after the day of week is considered a command to be run.
+
+.. option:: -o, options=LIST
+
+  Set comma separated `EXTRA OPTIONS`_ for all cron entries.
+
+.. option:: -E, --preserve-env
+
+  Export the current environment to be used for the command being executed
+  as part of the cron job.  Normally, the broker environment is used.
 
-      The :option:`--options` option allows a comma separated list of extra
-      options to be passed to the flux-cron service. See EXTRA OPTIONS below.
+.. option:: -d, --working-dir=DIR
+
+  Set the working directory to be set for the command being executed as part
+  of the cron job. Normally, the working directory of the broker is used.
+
+at
+--
+
+.. program:: flux cron at
+
+Run *command* at specific date and time described by *time*.
+
+.. option:: -o, options=LIST
+
+  Set comma separated `EXTRA OPTIONS`_ for all cron entries.
 
 .. option:: -E, --preserve-env
 
-      The :option:`--preserve-env` option allows the current environment to be
-      exported and used for the command being executed as part of the cron job.
-      Normally, the broker environment is used.
+  Export the current environment to be used for the command being executed
+  as part of the cron job.  Normally, the broker environment is used.
 
 .. option:: -d, --working-dir=DIR
 
-      The :option:`--working-dir` option allows the working directory to be set
-      for the command being executed as part of the cron job. Normally, the
-      working directory of the broker is used.
+  Set the working directory to be set for the command being executed as part
+  of the cron job. Normally, the working directory of the broker is used.
```

Comment:
Suggestion: Drop "to be set" here so it is just "Set the working directory for the command being executed..."

Reply:
Oh looking below maybe this is just a typo and you meant "Set the working directory _to be used_..."